graphql/types/SponsoredProduct.graphql

```diff
@@ -18,6 +18,11 @@ type SponsoredProduct {
   Sponsored campaign related information should be added here.
   """
   advertisement: Advertisement
+
+  """"
```

Comment:
There are 4 double quotes, it should be 3. I believe that's why build is failing.

Reply:
my bad!